packages/protocol-contracts/contracts/Zeta.non-eth.sol

```diff
@@ -29,12 +29,13 @@ contract ZetaNonEth is ERC20Burnable, ZetaErrors {
         tssAddressUpdater = tssAddressUpdater_;
     }
 
-    function updateTssAndConnectorAddresses(address newTssAddress, address newConnectorAddress) external {
+    function updateTssAndConnectorAddresses(address tssAddress_, address connectorAddress_) external {
         if (msg.sender != tssAddressUpdater) revert CallerIsNotTssUpdater(msg.sender);
-        if (newTssAddress == address(0)) revert InvalidAddress();
+        if (tssAddress_ == address(0)) revert InvalidAddress();
+        if (connectorAddress_ == address(0)) revert InvalidAddress();

```

Comment:
to keep a standard. zero validation together or not? has a slightly better performance in one condition